lego/apps/surveys/serializers.py

```diff
@@ -90,17 +90,11 @@ def create(self, validated_data):
             for answer in answers:
                 question = answer.pop('question')
 
-                if getattr(question, 'question_type') in [1, 2]:
-                    if getattr(question, 'question_type') is 1 and \
-                                    len(answer['selected_options']) > 1:
-                        raise exceptions.ValidationError('You cannot select multiple options for '
-                                                         'this type of question.')
-
-                    Answer.create(submission, question, **answer)
-                else:
-                    new_answer = Answer.objects.create(submission=submission, question=question,
-                                                       **answer)
-                    new_answer.save()
+                if getattr(question, 'question_type') is 1 and len(answer['selected_options']) > 1:
```

Comment:
Any reason for this `getattr` and not just `question.question_type is 1...`

Reply:
I believe that's what I did initially, but I think `question` at this point is some sort of Django manager or pointer or some such, and iirc `question.question_type` resulted in something like `Error: Django.RelatedObjectManager has no attribute "question_type"`.